upgrade_test.py

```diff
@@ -500,7 +500,8 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
                                      self.params.get('alternator_secret_access_key')))
             for table_name in alternator_table_names:
                 self.alternator.create_table(
-                    node=self.db_cluster.node_to_upgrade, schema=alternator_table_schema, table_name=table_name)
+                    node=self.db_cluster.node_to_upgrade, schema=alternator_table_schema,
+                    isolation=alternator.enums.WriteIsolation.FORBID_RMW, table_name=table_name)
```

Comment:
reminder to remove, since it was change only for reproduction of the issue.

Reply:
It has a separate comet :-)
Do you want me to separate the logic into a separate function?